Scarb.toml

```
@@ -8,6 +8,10 @@ edition = "2024_07"
 [dependencies]
 starknet = "2.8.5"
 snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.35.1" }
+assert_macros = ">=2.9.2"
```

```suggestion
assert_macros = "2.8.5"
```
Change the version for the test to pass